### ISSUE TITLE
Question: Saving checkpoints

### ISSUE THREAD

**Issue by mineshmathew** [SakuraRiven/EAST#53]
@SakuraRiven  For how many epochs did you train the pretrained model ? 
And how did you decide which checkpoint to use? based on the average loss per epoch?
